modules/gltf/src/gltf/gltf-parser.js

```diff
@@ -365,7 +402,21 @@ export default class GLTFParser {
     if (createImages) {
       image.image = this.glbParser.getImage(image);
     } else {
-      image.getImageAsync = () => this.glbParser.getImageAsync(image);
+      image.getImageAsync = () => {
```

Comment:
Do we still need this code? Aren't we using async textures now?

if keep, use `async/await`
```
await getImageAsync
```